acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/publickeys/YubiHsmKeysAcceptanceTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.tests.publickeys;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static tech.pegasys.web3signer.core.util.IdentifierUtils.normaliseIdentifier;
+
+import tech.pegasys.signers.secp256k1.EthPublicKeyUtils;
+import tech.pegasys.teku.bls.BLSSecretKey;
+import tech.pegasys.web3signer.core.signing.KeyType;
+import tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import io.restassured.response.Response;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.web3j.crypto.Credentials;
+
+@Disabled("Requires physical access to YubiHSM device and SDK installed")
+public class YubiHsmKeysAcceptanceTest extends KeyIdentifiersAcceptanceTestBase {
+  // update following path with exact location of yubi pkcs11 module
+  private static final String MODULE_PATH =
```

Comment:
should this just an env var? I.e. don't leave behind any static strings?

Reply:
Hmm ... env variable would work as well.